taskcluster/ci/build-bundle/kind.yml

```diff
@@ -37,13 +37,6 @@ jobs:
         include-nightly-version: true
```

Comment:
Nit: Let's add `include-shippable-secrets: true` here too

Reply:
Ah yes, of course.